src/getTime.js

```diff
@@ -18,7 +18,10 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const regEx = /\b((([0-1][0-9])|(2[0-3])):([0-5][0-9]))\b/;
+  const timeFromStr = str.match(regEx);
```

Comment:
метод возвращает массив совпадений, называй соответствующе
```suggestion
  const matches = str.match(regEx);
```